lte/cloud/go/services/subscriberdb/digest_test.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package subscriberdb_test
+
+import (
+	"fmt"
+	"testing"
+
+	"magma/lte/cloud/go/lte"
+	"magma/lte/cloud/go/serdes"
+	lte_models "magma/lte/cloud/go/services/lte/obsidian/models"
+	lte_test_init "magma/lte/cloud/go/services/lte/test_init"
+	"magma/lte/cloud/go/services/subscriberdb"
+	"magma/lte/cloud/go/services/subscriberdb/obsidian/models"
+	"magma/orc8r/cloud/go/orc8r"
+	"magma/orc8r/cloud/go/services/configurator"
+	configurator_test_init "magma/orc8r/cloud/go/services/configurator/test_init"
+	"magma/orc8r/cloud/go/storage"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	iterationCount = 50
+	// subscriberCount is the number of test subscribers created to test deterministic streaming of subscribers.
+	// You can test for deterministic digest generation over multiple pages of subscriber data by reducing
+	// the maximum page load size for configurator to less than this value.
+	subscriberCount = 50
```

Comment:
- Don't put this type of const in the pkg-global namespace -- just put it directly in each relevant test
- Go ahead and enact that test you mention, in a separate test here. Can add a new `configurator_test_init.StartTestServiceWithOptions` method that takes, as a parameter, a max page size

Reply:
Actually, just realized that the `testServiceMaxLoadSize` is 10 for `configurator`, which means the original test should already be testing multiple pages @hcgatewood